packager/rbi_gen.cc

```diff
@@ -1167,6 +1164,23 @@ class RBIExporter final {
         return lookupFQN(gs, fullName).asClassOrModuleRef();
     }
 
+    string buildPackageDependenciesString(const core::GlobalState &gs) {
+        vector<string> packageNames;
+        packageNames.reserve(referencedPackages.size());
+        absl::c_transform(referencedPackages, back_inserter(packageNames),
+                          [&gs](const auto &p) -> string { return p.first.show(gs); });
+        fast_sort(packageNames, [](const auto &lhs, const auto &rhs) -> bool { return lhs < rhs; });
```

Comment:
Are FileRef IDs stable?

Reply:
Do we need to sort by the file path instead?

Reply:
i guess it looks like this is not the first place we're sorting on these IDs, and the `operator<` definition even exists so maybe it's not worth worrying about

Reply:
Given that we sort by `FileRef` for determinism purposes in e.g. resolver:

https://github.com/sorbet/sorbet/blob/10d09b92d34d832bef454f0436cf0b35a7003cb2/resolver/resolver.cc#L1524-L1539

(and a number of other places), I think this is OK.  It does mean that the output is not stable if you change the order files are given on the command line, but I think that is mostly OK?